lib/src/widget/text_field/app_text_field.dart

```diff
@@ -39,9 +39,7 @@ class AppTextField extends StatelessWidget {
   Widget build(BuildContext context) {
     final linden = UnterDenLinden.getLinden(context);
 
-    final textFieldStyle = linden.dimen.isSmallScreen
-        ? linden.styles.textInputTextSmall
-        : linden.styles.textInputText;
+    final textFieldStyle = linden.styles.textInputText;
```

Comment:
❓ The check `isSmallScreen` is not needed anymore ?

Reply:
That's correct. We don't have small input fields in new designs.